eng/Versions.props

```diff
@@ -7,13 +7,13 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>1</PatchVersion>
```

Comment:
The `PatchVersion` needs to be 0 right?

Reply:
For package precedence purposes, this should match the current patch level in the servicing releases.

Reply:
So everytime we bump the `PatchVersion` on `release/6.0` we will need to bump it on this branch.

Reply:
Yeah. Or just merge from release/6.0. That is also important to avoid regressing the maui branch.

Reply:
The number is taken from release/6.0 - should it be lower than that branch? https://github.com/dotnet/runtime/blob/release/6.0/eng/Versions.props#L8

Reply:
> Yeah. Or just merge from release/6.0. That is also important to avoid regressing the maui branch.

Makes sense.